starterkits/basic/packages/primary-components/atoms/input/a-input.html.twig

```diff
@@ -0,0 +1,2 @@
+<span class="a-input__label">{{ content.label }}</span>
```

Comment:
maybe manage labels to another level since it is not only input related?

Reply:
issue created https://github.com/skilld-labs/kaizen/issues/98